src/components/ComputerBoard.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from 'react';
+import { ComputerBoardContext } from '../context/ComputerBoardContext';
+import Cell, { CellInterface } from './Cell';
+
+const ComputerBoard: React.FC = function () {
+  const computerBoardCtx = useContext(ComputerBoardContext);
+
+  const gridMap = computerBoardCtx?.grid.map((row, idx) => (
+    row.map((cell : CellInterface, jdx : number) => (
+      <Cell
```

Comment:
We should be passing in a unique `key` prop so that React knows about each of these `Cells`.

More info: https://reactjs.org/docs/lists-and-keys.html

Reply:
Fixed: Using the current date/time as the key